Development.md

```diff
@@ -9,8 +9,8 @@ There are a few places where version numbers appare:
 |Core   |lonewolf/content/version.ts:3                   |`x.y.z`|Yes |
 |Web    |lonewolf-web/src-platform/content/version.ts:3  |`x.y.z`|Yes |
 |Desktop|lonewolf-tauri/src-platform/content/version.ts:3|`x.y.z`|Yes |
-|Desktop|lonewolf-tauri/src-tauri/tauri.conf.json:11     |`x.y.z`|Yes |
 |Desktop|lonewolf-tauri/src-tauri/Cargo.toml:3           |`0.0.0`|No  |
+|Desktop|lonewolf-tauri/src-tauri/tauri.conf.json:11     |`0.0.0`|No  |
```

Comment:
I'm trying to use `appVersion` instead: https://github.com/tauri-apps/tauri-action#project-initialization

To be honest I'm just trying a few things, but since I can't push tags on your repo I created a branch in this repo https://github.com/aichingm/lonewolf/tree/github-actions-windows-build

Reply:
Ok `appVersion` seams to only work if there is no tauri project already

Reply:
I fixed that by reading the version from the tag and adding it to the json file.